Big-Spender/readme.md

```diff
@@ -43,17 +43,22 @@ You are working with Claire and Farnoosh, who are trying to complete a missing r
 
 **You:** I can try. What kind of data are you looking for exactly?
 
+big-spender=# select * from spends where date between '2021-01-01' and '2021-12-31';    
+
 **Farnoosh:** We need to find out the transactions between £30,000 and £31,000. Could you help us write a query for that?
 
 **You:** Absolutely. Here's the SQL query you need:
+big-spender=# select * from spends where date between '2021-01-01' and '2021-12-31' and amount  between 30000 and 31000;    
 
 ```sql
 INSERT YOUR QUERY HERE
+big-spender=# select * from spends where date between '2021-01-01' and '2021-12-31' and amount  between 30000 and 31000;    
 ```
 
 **Claire:** That's great, thanks. Hey, what about transactions that include the word 'fee' in their description?
 
 **You:** Does case matter?
+select * from spends where description like '%fee%';   
```

Comment:
what is this is FEE? how would this query handle it? Would it show the results? You might need to make this case-insensitive? What would we do to do this?